BackofficeBundle/Resources/config/field_type.yml

```diff
@@ -111,9 +111,9 @@ parameters:
                     required: true
         date:
             label: open_orchestra_backoffice.form.field_type.custom_type.date
-            type: date
+            type: date_picker
```

Comment:
I think it should be better to have only one date component, with several options to choose the rendering